src/cmakeFindModules/OpenMVGConfig.cmake.in

```diff
@@ -83,7 +83,7 @@ set(THREADS_PREFER_PTHREAD_FLAG ON)
 find_package(Threads REQUIRED)
 
 # Import exported OpenMVG targets
-include(${CURRENT_CONFIG_INSTALL_DIR}/OpenMVGTargets.cmake)
+include(${CURRENT_ROOT_INSTALL_DIR}/lib/openMVG-targets.cmake)
```

Comment:
This one is wrong. should be something like `${CURRENT_CONFIG_INSTALL_DIR}/CMakeFiles/Export/share/openMVG/cmake`